lib/workers/repository/finalise/repository-statistics.ts

```diff
@@ -42,11 +42,12 @@ export function runRenovateRepoStats(
 
 function branchCacheToMetadata({
   branchName,
-  sha: branchSha,
   baseBranch,
   baseBranchSha,
   automerge,
   isModified,
+  sha: branchSha,
```

Comment:
Why reorder if they're not alphabetically sorted anyway?

Reply:
only to group together literal renames, aesthetic wise looks better to me

```ts
function branchCacheToMetadata({
  branchName,
  baseBranch,
  baseBranchSha,
  automerge,
  isModified,
  sha: branchSha,
  pristine: isPristine,
}: BranchCache): BranchMetadata {}
```

vs

```ts
function branchCacheToMetadata({
  branchName,
  sha: branchSha,
  baseBranch,
  baseBranchSha,
  automerge,
  isModified,
  pristine: isPristine
}: BranchCache): BranchMetadata {}
```

Reply:
now sorted as well which ticks both boxes